swagger/v1/openapi.json

```diff
@@ -167,6 +167,257 @@
         }
       }
     },
+    "/api/compliance/rule_results": {
```

Comment:
This _is_ autogenerated, right? Do we have instructions in the readme for how to generate the openapi docs?

Reply:
Correct: I use this: https://github.com/domaindrivendev/rswag `rake rswag:specs:swaggerize`

Reply:
Do you think it'd be a good idea to mention that somewhere in the Readme?